pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -596,7 +596,12 @@ func calcBucketBound(bucketOptions cloudMonitoringBucketOptions, n int) string {
 	case bucketOptions.ExponentialBuckets != nil:
 		bucketBound = strconv.FormatInt(int64(bucketOptions.ExponentialBuckets.Scale*math.Pow(bucketOptions.ExponentialBuckets.GrowthFactor, float64(n-1))), 10)
 	case bucketOptions.ExplicitBuckets != nil:
-		bucketBound = fmt.Sprintf("%g", bucketOptions.ExplicitBuckets.Bounds[n])
+		if n < len(bucketOptions.ExplicitBuckets.Bounds) {
+			bucketBound = fmt.Sprintf("%g", bucketOptions.ExplicitBuckets.Bounds[n])
+		} else {
+			lastBound := bucketOptions.ExplicitBuckets.Bounds[len(bucketOptions.ExplicitBuckets.Bounds)-1]
+			bucketBound = fmt.Sprintf("%g+", lastBound)
+		}
```

Comment:
Could we simplify with ternary operator:

```suggestion
        i := n < len(bucketOptions.ExplicitBuckets.Bounds) ? n : len(bucketOptions.ExplicitBuckets.Bounds)-1
		bucketBound = fmt.Sprintf("%g+",bucketOptions.ExplicitBuckets.Bounds[i])
```

Reply:
there is no ternary operator in Go :) Also, the  `+` of `%g+` is only added for the case in which the index is outside bounds